README.md

```diff
@@ -212,14 +220,16 @@ Additional notes for building nextpnr
 The following runs a debug build of the iCE40 architecture without GUI, without Python support, without the HeAP analytic placer and only HX1K support:
 
 ```
-cmake . -DARCH=ice40 -DCMAKE_BUILD_TYPE=Debug -DBUILD_PYTHON=OFF -DICE40_HX1K_ONLY=1
+mkdir -p build && cd build
+cmake .. -DARCH=ice40 -DCMAKE_BUILD_TYPE=Debug -DBUILD_PYTHON=OFF -DICE40_HX1K_ONLY=1
```

Comment:
The `ICE40_HX1K_ONLY` flag is no longer recognized by the build system (not sure since when, I didn't remove it a week ago...)

Reply:
Fixed this